### DISCUSSION TITLE
Disable TTS?

### DISCUSSION THREAD

**Discussion by bennysweb** [Thyraz/Sonos-Kids-Controller#18] [Q&A]
Dear all,
is it possible to deactivate the text-to-speech? My daughter is 7 years old, has good reading skills and is confused about this ;-)
Thanks in advance
Benny

**Comment by imhofroger** [comment]
Hi Benny
You are right.
I have added the function to deactivate and adjust the text-to-speech function.
This is done using the config.json file.
It is currently in a pull request.
regards,
Roger
